docs/customization.md

```diff
@@ -236,3 +237,36 @@ After creating the class, set the `$userProvider` property in **app/Config/Auth.
 ```php
 public string $userProvider = \App\Models\UserModel::class;
 ```
+
+## Custom Login Identifier
+
+If your application has a need to use something other than `email` or `username`, you may specify any valid column within the `users` table that you may have added. This allows you to easily use phone numbers, employee or school IDs, etc as the user identifier. You must implement the following steps to set this up:
+
+This only works with the Session authenticator.
+
+1. Create a [migration](http://codeigniter.com/user_guide/dbmgmt/migration.html) that adds a new column to the `users` table.
+2. Edit `app/Config/Auth.php` so that the new column you just created is within the `$validFields` array.
+
+```php
+public array $validFields = [
+    'employee_id'
+];
+```
+
+If you have multiple login forms on your site that use different credentials, you must have all of the valid identifying fields in the array.
+
+```php
+public array $validFields = [
+    'email',
+    'employee_id'
+];
+```
```

Comment:
```suggestion
    ```php
    public array $validFields = [
        'employee_id'
    ];
    ```

    If you have multiple login forms on your site that use different credentials, you must have all of the valid identifying fields in the array.

    ```php
    public array $validFields = [
        'email',
        'employee_id'
    ];
    ```
    > **Warning**
    > It is very important for security that if you add a new column for identifier you must write a new **Validation Rules** and then set it using the [custom-validation-rules](https://github.com/codeigniter4/shield/blob/develop/docs/customization.md#custom-validation-rules) description.
```

Reply:
![Screenshot 2023-02-08 043740](https://user-images.githubusercontent.com/9530214/217403003-f0c28a32-f069-49ce-b1ca-606772648312.jpg)

**lonnieezell** , I know **paulbalandan**  was concerned about security, I added a warning, you can change the text.

Reply:
That text sounds pretty good to me, actually. Thanks for pointing that step out.